internal/clientconn/listener.go

```diff
@@ -82,9 +82,45 @@ func NewListener(opts *NewListenerOpts) *Listener {
 	}
 }
 
+// Wait blocks until all of the expected listeners have started or returned the error.
+func (l *Listener) Wait() {
```

Comment:
ooooooof…

Reply:
Do we have any better way of implementing that?

Reply:
Hopefully. But I don't know what is that yet

Reply:
#4404